example/components/events.js

```diff
@@ -20,7 +20,7 @@ export default class EventsExample extends Component<{}, State> {
     },
   }
 
-  mapRef = createRef<Map>()
```

Comment:
It is correct. The `<Map>` is defining the Flow type of the ref.  https://stackoverflow.com/questions/50076176/how-to-use-flow-with-react-createref